### DISCUSSION TITLE
Support for lightweight-chart in .Net core WebView2 control

### DISCUSSION THREAD

**Discussion by rajendrasp** [tradingview/lightweight-charts#762] [Q&A] [answered]
Hi,
I am trying to have this lightweight-chart library into my .Net Core app using WebView2 control (https://docs.microsoft.com/en-us/microsoft-edge/webview2/how-to/javascript). I can run Javascript file into this control. But could not figure out if it is possible to render the chart using this method. Could you help me figure it out? How can we use lightweight-chart functionality in js file if library is not installed using npm? Or is it simply not possible to do it this way? Also do you have a library I can use in a desktop app?
Thanks

**Comment by timocov** [comment, marked as answer]
> How can we use lightweight-chart functionality in js file if library is not installed using npm? 

You can 1) use bundlers like webpack, rollup, esbuild to bundle all your files into single js file 2) use standalone version of the library which is in the every version of lightweight-charts.

> Also do you have a library I can use in a desktop app?

You still can use lightweight-charts with webview, but we don't have "a native" version though.

**Comment by youngpimpy** [comment]
> Hi,
> 
> I am trying to have this lightweight-chart library into my .Net Core app using WebView2 control (https://docs.microsoft.com/en-us/microsoft-edge/webview2/how-to/javascript). I can run Javascript file into this control. But could not figure out if it is possible to render the chart using this method. Could you help me figure it out? How can we use lightweight-chart functionality in js file if library is not installed using npm? Or is it simply not possible to do it this way? Also do you have a library I can use in a desktop app?
> 
> Thanks

So u installed successfully?
